idl/github.com/uber/tchannel/meta.thrift

```diff
@@ -22,7 +39,7 @@ struct VersionInfo {
 }
 
 service Meta {
-    HealthStatus health()
+    HealthStatus health(1: HealthRequest hr)
```

Comment:
I think it will be wire compatible, and from this PR that is probably enough.

I know that Go will fail to compile when a new param is added, e.g. you need to manually add the param to the handlers implementing the interface.